src/components/wallets/CustomerWalletList.tsx

```diff
@@ -0,0 +1,119 @@
+import { forwardRef, MutableRefObject } from 'react'
+import { gql } from '@apollo/client'
+
+import { Button, InfiniteScroll, Typography } from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { SectionHeader, SideSection } from '~/styles/customer'
+import { GenericPlaceholder } from '~/components/GenericPlaceholder'
+import ErrorImage from '~/public/images/maneki/error.svg'
+import { useGetCustomerWalletListQuery } from '~/generated/graphql'
+
+import { AddWalletToCustomerDialogRef } from './AddWalletToCustomerDialog'
+import { WalletAccordion, WalletAccordionSkeleton } from './WalletAccordion'
+
+gql`
+  query getCustomerWalletList($customerId: ID!, $page: Int, $limit: Int) {
+    wallets(customerId: $customerId, page: $page, limit: $limit) {
+      metadata {
+        currentPage
+        totalPages
+      }
+      collection {
+        id
```

Comment:
This should be a fragment in WalletAccordion where it's used

Reply:
Done, sorry I didn't at first 😅